### PR TITLE
bug: unmatched div tag in menu-item.template.ts

### PR DESCRIPTION
**Pull request by kamalrohra**
# Pull Request

## 📖 Description

There was an unmatched div in on [line](https://github.com/microsoft/fast/blob/6091f89a6a0b8bf3baa4cd4ed2aae1078ba9865b/packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts#L58) , This pull request is to address the same bug 

### 🎫 Issues

This pull request is to address the issue [#6840](https://github.com/microsoft/fast/issues/6840)

### General

- [x] I have included a change request file using `$ yarn change`
- [x] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [x] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

File: packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts [hunk 1]
```diff
@@ -21,72 +21,76 @@ export function menuItemTemplate<T extends FASTMenuItem>(
...
+            aria-haspopup="${x => (x.hasSubmenu ? "menu" : void 0)}"
+            aria-checked="${x => (x.role !== MenuItemRole.menuitem ? x.checked : void 0)}"
+            aria-disabled="${x => x.disabled}"
+            aria-expanded="${x => x.expanded}"
+            @keydown="${(x, c) => x.handleMenuItemKeyDown(c.event as KeyboardEvent)}"
+            @click="${(x, c) => x.handleMenuItemClick(c.event as MouseEvent)}"
+            @mouseover="${(x, c) => x.handleMouseOver(c.event as MouseEvent)}"
+            @mouseout="${(x, c) => x.handleMouseOut(c.event as MouseEvent)}"
+        >
+            <div>
```
Comment (bheston) [inline packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts:34]: The div is an unnecessary wrapper element and should be removed. If the item is in checkbox or radio mode there will be the `input-container` element which serves the same purpose.

**Review by bheston — CHANGES_REQUESTED** [review]

File: packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts (hunk 1 above)
Comment (kamalrohra) [reply packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts]: Okay, understood I have pushed a new commit, thanks for guidance!

**Review by scomea — APPROVED** [review]

**Comment by bheston** [comment]
Please run `yarn change` and push your branch again. I'll merge this in as soon as I see your update.

**Comment by KingOfTac** [comment]
> Please run `yarn change` and push your branch again. I'll merge this in as soon as I see your update.

Looks the CLA needs to be accepted as well.

**Pull request closed without merging**